use-api.md

```diff
@@ -0,0 +1,144 @@
+|start_date|Filter results by start date.|start_date=1932|
+|end_date|Filter results by start date.|end_date=1975|
+|category|Filter results by category term, including `person`, `collection`, or `organization`|category=person|
+|subject|Filter results by subject/topic term|subject=public health|
+|creator|Filter results by creator. Creators are the people, organizations, and families responsible for creating the records.|creator=adams, lillian|
+|online|Filter results by objects that are digital and available to view online.|online=true|
+|genre|Filters results by genre/format term, including `documents`, `photographs`, `moving image`, and `audio`.|genre=moving image|
+|query|Query for full-text search|query=yellow fever|
+|sort|Sort results by title, start_date, end_date, or type. By default the named property will be sorted ascending. Descending order can be achieved by appending an en dash (`-`) to the start of the property.|sort=title|
+
+## Example queries
+1. Use the `/search` endpoint to return the number of search matches for the query term "agriculture" that are in collections and have been categorized as photographs with dates between 1940 and 1950:
+
+```
+https://api.rockarch.org/search?&query=agriculture&category=collection&genre=photographs&start_date__gte=1940&end_date__lte=1950
+```
+
+**Note**: Appending `__gte` and `__lte` to the date parameters function as `greater than or equal to` and `less than or equal to`, allowing us to include any start and end dates in this decade instead of limiting ourselves to specific start and end dates. Similarly, `gt`= greater than and `ls`= less than.
+
+2. Use the `/minimap` endpoint to return collections and objects with search hits for the query term "agriculture" within the Ford Foundation records collection:
+
+```
+https://api.rockarch.org/collections/2HnhFZfibK6SVVu86skz3k/minimap?query=agriculture
+```
+
+3. Write a Python script using the [rac_api_client](https://pypi.org/project/rac-api-client/) to identify the creators of collections that contain keyword search matches for "green revolution". The `/search` endpoint performs search queries across agents, collections, objects, and terms.
+
+Creators are the people, organizations, or families responsible for creating the records.
+Terms are controlled values describing topics, geographic places, or record formats.
+
+```python
+# import rac_api_client module
+from rac_api_client import Client
+
+# search across agents, collections, objects and terms for "green revolution"
+client = Client()
+response = client.get("/search", params={"query": "green revolution"})
+
+# compile a list of creators of collections that contain search matches for the query
+creator_list = []
+
+for collection in response["results"]:
+  for creator in (collection["creators"]):
+    if creator not in creator_list:
+      creator_list.append(creator)
+
+# print list of creators
+print(creator_list)
```

Comment:
This could be done in a single line with nested list comprehension BUT that is probably more confusing than what you're doing here. However, rather than checking whether or not a creator is in the list already (line 136) I think I'd just call `set` on the `creator_list`, which will remove duplicates. If you want a list you can do `list(set(creator_list))` which will convert the set back to a list.